tls-client/main.cpp

```diff
@@ -148,9 +151,7 @@ class HelloHTTPS {
         _error = false;
         _disconnected = false;
         _request_sent = false;
-        /* Fill the request buffer */
-        _bpos = snprintf(_buffer, sizeof(_buffer) - 1, "GET %s HTTP/1.1\nHost: %s\n\n", path, HTTPS_SERVER_NAME);
-
+        
```

Comment:
There are whitespace characters on this empty line, could you please remove them?